lib/cheets.rb

```diff
@@ -0,0 +1,20 @@
+require './app'
+require 'data_mapper'
+
+DataMapper.setup(:default, 'postgres://localhost/chitter')
+
+class Cheets
+
+  include DataMapper::Resource
+  property :id,         Serial
+  property :body,       String
+  property :created_at, DateTime
+
+  DataMapper.finalize
+  DataMapper.auto_migrate!
+  DataMapper.auto_upgrade!
+
+  Cheets.create body: 'Test Cheet One', created_at: Time.now
```

Comment:
This feels like test code, and should really be in a spec or a spec_helper - test and production code should be kept separately.